src/DbInterface.cpp

```diff
@@ -234,7 +234,7 @@ void DbInterface::handlePostSwitchCause(
         boost::posix_time::ptime time
 )
 {
-    MUXLOGDEBUG(boost::format("%s: post switch cause %s") %
+    MUXLOGWARNING(boost::format("%s: post last switch cause %s") %
```

Comment:
Is this too verbose? What's the reason for the change?

Reply:
> Is this too verbose? What's the reason for the change?

It will add one extra log for each switchover. 

Telemetry won't onboard switch cause data to Kusto anytime soon, so I wanted to have it in syslog for now.